wikimedia/wikipedia-ios
# Native Editor - Add heading syntax highlighting

**Phabricator:** 
https://phabricator.wikimedia.org/T348073

### Notes
This adds syntax highlighting for headings.

### Test Steps
1. Go to an article on Staging scheme. Tap the "Edit" button in an article navigation view.
2. Confirm headings syntax highlight as expected. Edit the number of equal signs on an editing. Confirm font size grows and shrinks as expected.
3. Change font size in popover. Confirm font size grows and shrinks on headings in relation to other text as expected.
4. Change theme in popover. Confirm heading colors change as expected.
5. Toggle off syntax highlighting. Confirm syntax highlighting toggles as expected.

### Screenshots/Videos
